tdr_preingest.tf

```diff
@@ -2,6 +2,8 @@ locals {
   tdr_preingest_name              = "${local.environment}-dr2-preingest-tdr"
   tdr_aggregator_name             = "${local.tdr_preingest_name}-aggregator"
   tdr_aggregator_queue_arn        = "arn:aws:sqs:eu-west-2:${data.aws_caller_identity.current.account_id}:${local.tdr_aggregator_name}"
+  tdr_aggregator_batching_window  = 300
+  tdr_aggregator_batch_size       = 10000
```

Comment:
Is this Bytes or Kilobytes,  Maybe "bytes" or "kb" could be appended to the name?

Reply:
It's neither, it's the number of items in the queue. AWS call it batch size to I'll keep it so they match.